src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    public static final int OPERATION_TYPE_INDEX = 0;
+    public static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFile(fromFileName).split(" ");
+        String report = createReport(data);
+        writeData(report, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder dataBuilder = new StringBuilder();
+        String value;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            value = bufferedReader.readLine();
+            while (value != null) {
+                dataBuilder.append(value).append(" ");
```

Comment:
You can try to use List instead of StringBuilder, to avoid splitting 
```suggestion
                result.add(value);
```

Reply:
Or you can try to use Files.readAllLines method